themes/startwords/assets/scss/article/_figure.scss

```diff
@@ -9,6 +9,7 @@ body.article .text-container > p > img {
 
 body.article figure {
     margin: rem(30px) 0;
+    page-break-inside: avoid;
```

Comment:
This makes sense, correct?

Reply:
yes

Reply:
although it might need to be in the print styles?

Reply:
It certainly works here, but if you want all print-specific styling within a `@media print` block, let me know!

Reply:
I moved it to the print styles file — I think it will be helpful when reviewing them or revising them to have them more centralized, in case we need to make adjustments.